pom.xml

```diff
@@ -137,7 +143,7 @@
         <configuration>
           <instructions>
             <_exportcontents>
-              io.cdap.plugin.*;
+              io.cdap.plugin.cobol.*;
               com.google.common.*;
```

Comment:
since this is a transform, I'm pretty sure com.google.common.* and com.legstar.* are not needed, let's remove them.

Reply:
Done. 

Also I'm curious why these imports may be sometimes necessary, and why not for transform.

Reply:
The exports are always required for the plugin classes themselves, which is why io.cdap.plugin.cobol.* is being exported here. Otherwise, it basically only for sources to export their InputFormat class and sinks to export their OutputFormat class.

The reason is that the MapReduce/Spark program reads from its configuration that it needs to instantiate InputFormat class 'xyz.myclass'. So the program's classloader needs access to those classes in order to instantiate them. For transforms, the program just uses the Transform API defined in cdap-etl-api and doesn't need try to do anything with any of the classes within the plugin itself. 

See https://docs.cask.co/cdap/6.0.0/en/developer-manual/advanced/class-loading.html for a more detailed explaining of CDAP's classloading model.